packages/cross_file/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.3.4+1
+
+* Removes the following deprecated APIs from the web implementation:
```

Comment:
I think it's fine to be more vague here. Careful adding details that don't help any end user.

Reply:
Added lightness to the changelogs!